### PR TITLE
Small fixes to README.md

### PR DESCRIPTION
**Pull request by dlunde**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ creating languages and generating efficient compilers.
 Before you test the Miking system, you need to install
```
Comment (david-broman) [inline README.md]: I have rewritten this part in PR #107. I would prefer the other changes.

File: README.md [hunk 2]
```diff
@@ -48,6 +48,7 @@ user, issue:
 >>> make install
 ```
```
Comment (david-broman) [inline README.md:50]: Looks good.

File: README.md [hunk 3]
```diff
@@ -721,7 +722,7 @@ Miking that does not need these external dependencies.
 One of the external dependencies is Sundials, a numerical library for
```
Comment (david-broman) [inline README.md:722]: OK!

**Review by david-broman — COMMENTED** [review]
Good additions!

File: README.md (hunk 1 above)
Comment (david-broman) [reply README.md]: Hence, it would be good if you reverted this sentence, so that we can rebase without conflicts.

Comment (dlunde) [reply README.md]: Done

**Pull request merged**